src/components/molecules/FirstBootomSection.js

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+
+const FirstBootomSection = ({
```

Comment:
What are your thoughts on adjusting the name for this component slightly?
```suggestion
const FirstFooterSection = ({
```